migration/migrating_3_4/configuring-replication-repository-3-4.adoc

```diff
@@ -1,21 +1,25 @@
-[id='configuring-replication-repository_{context}']
+[id='configuring-replication-repository']
 = Configuring a replication repository
 include::modules/common-attributes.adoc[]
 :context: migrating-3-4
 :migrating-3-4:
 
 toc::[]
 
-You must configure an object storage to use as a replication repository. The Cluster Application Migration tool copies data from the source cluster to the replication repository, and then from the replication repository to the target cluster, using either the xref:../../migration/migrating_3_4/configuring-replication-repository.adoc#file-system-copy-method_{context}[file system] or the xref:../../migration/migrating_3_4/configuring-replication-repository.adoc#snapshot-copy-method_{context}[snapshot] data copy method.
+You must configure an object storage to use as a replication repository. The Cluster Application Migration tool copies data from the source cluster to the replication repository, and then from the replication repository to the target cluster, using either the xref:../../migration/migrating_3_4/configuring-replication-repository-3-4.adoc#file-system-copy-method_{context}[file system] or the xref:../../migration/migrating_3_4/configuring-replication-repository-3-4.adoc#snapshot-copy-method_{context}[snapshot] data copy method.
 
 The following storage providers are supported:
 
-* xref:migration-configuring-mcg_{context}[Multi-Cloud Object Gateway (MCG)]
-* xref:migration-configuring-aws-s3_{context}[Amazon Web Services (AWS) S3]
-* xref:migration-configuring-gcp_{context}[Google Cloud Provider (GCP)]
-* xref:migration-configuring-azure_{context}[Microsoft Azure]
+* xref:migration-configuring-mcg_migrating-3-4[Multi-Cloud Object Gateway (MCG)]
```

Comment:
By convention we've been including `../../migration/migrating_3_4` for these, as you've done in other places.